### PR TITLE
Recipe for pla rad prik.

### PR DESCRIPTION
**Pull request by mookerji**
/cc @zmagg

### CODE REVIEW

File: _posts/meat/2018-08-05-pla-rad-prik.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Pla Rad Prik
+category: meat
+---
+
+[Pla Rad Prik][pla_rad_prik] is a Thai dish: whole-fried, meaty white fish with
+chili sauce, usually a centerpiece entree. It's pretty common to find red
+snapper (delicious!) in Thai restaurants in the US; farmed or less over-fished
+alternatives like grouper, pompano, white perch, haddock, etc. probably work as
```
Comment (zmagg) [inline _posts/meat/2018-08-05-pla-rad-prik.md]: nit: haddock would be a bad substitute.

File: _posts/meat/2018-08-05-pla-rad-prik.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+well. Ideally, whatever fish is sized appropriately for your wok.
+
+- For the sauce, food process:
```
Comment (zmagg) [inline _posts/meat/2018-08-05-pla-rad-prik.md]: Is there an alternative to using a food processor here?

File: _posts/meat/2018-08-05-pla-rad-prik.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  * 1/2 cup - garlic cloves
+  * 1/2 cup - small, red Thai chilies (de-seeded)
+
+  Also set aside:
+  * 1/2 cup - tamarind paste (prepared from pulp, if possible).
```
Comment (zmagg) [inline _posts/meat/2018-08-05-pla-rad-prik.md]: what do you mean by 'prepared from pulp'? can you link to photos of the pulpy kind and the liquidy kind, this was not obvious to me until recently.

File: index.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ title: House Recipes
 
 {% include list.html category="slowbread" %}
 
-### Meat
+### Meat and Fish
```
Comment (zmagg) [inline index.md]: lol

File: _posts/meat/2018-08-05-pla-rad-prik.md (hunk 1 above)
Comment (mookerji) [reply _posts/meat/2018-08-05-pla-rad-prik.md]: Fixed

File: _posts/meat/2018-08-05-pla-rad-prik.md (hunk 2 above)
Comment (mookerji) [reply _posts/meat/2018-08-05-pla-rad-prik.md]: Fixed

File: _posts/meat/2018-08-05-pla-rad-prik.md (hunk 3 above)
Comment (mookerji) [reply _posts/meat/2018-08-05-pla-rad-prik.md]: Fixed

**Pull request merged**